src/main/groovy/SpringSessionGrailsPlugin.groovy

```diff
@@ -78,14 +69,6 @@ class SpringSessionGrailsPlugin extends Plugin {
                 }
             }
 
-            sessionRedisTemplate(RedisTemplate) { bean ->
```

Comment:
Current version of Spring Session Redis lib created does not look for RedisTemplate in the context, it creates it's own injecting default serializer (https://github.com/RxLogix/spring-session/pull/2/files#diff-8a684a17bb03653ab91cf2b33fe0e44d9ee207935cf8f5489a4f5d6fc41b7abfR18).